src/doc/contrib/src/team.md

```diff
@@ -0,0 +1,157 @@
+- Voting and decision making (RFCs, major changes)
+- Access to private communications related to team management and security discussions
+
+[coc]: https://www.rust-lang.org/policies/code-of-conduct
+
+### Meetings
+
+The team meets on a weekly basis on a video chat.
+If you are interested in participating, feel free to contact us on [Zulip].
+
+### Becoming a member
+
+A contributor can become a member of the Cargo Team by requesting a review or being nominated by one of the existing members.
+They can be added by a unanimous vote from the team.
+
+Contributors who wish to join the team should exhibit an interest in carrying the design principles of Cargo and participate in some of the activities listed above in [Membership Expectations](#membership-expectations).
+
+Members may leave at any time, preferably by letting the team know ahead of time.
+
+### Leadership
+
+The team leader is responsible for coordinating the team and providing a contact point with other teams.
+The leader is selected by a vote of the existing members.
+
+## Decision process
+
+The team uses a consensus-driven process for making decisions ranging from new features and major changes to management of the team itself.
+The degree of process is correlated with the degree of change being proposed:
+
+- Bug fixes, refactorings, documentation updates, and other small changes are usually delegated to a single team member (who is not the author) to approve based on their judgement.
+  Team members may also solicit feedback from other members or the whole team for any change should they want to gather other perspectives from the team.
+
+  Some examples of what this might cover are:
+  - Bug fixes that do not introduce backwards-incompatible changes, and adhere to Cargo's expected behavior.
+  - Addition of new warnings, or other diagnostic changes.
+  - New or updated documentation.
+  - Localized refactorings (that is, those that do not have a significant, wide-ranging impact to the usage and maintenance of the codebase).
+  - Minor or planned changes to Cargo's console output.
+  - Beta backports that clearly address a regression, and are expected to be low-risk.
+  - Development of a previously approved unstable feature that matches the expected development of that feature.
+
+- Small features or changes, large refactorings, or major changes to Cargo's codebase or process require an approval by the team via consensus.
+  These decisions can be done via the FCP process of [rfcbot], or in an ad-hoc manner such as during a team meeting.
+  rfcbot FCP requests do not require waiting for the 10-day feedback window if there is a complete team consensus, as this process is mostly aimed at polling the team rather than publicly soliciting feedback.
+  Though, public feedback is welcome at any time.
+
+  Some examples of what this might cover are:
+  - Addition of a new, minor command-line argument, or an addition of an option to an existing one.
```

Comment:
I wonder which category adding a new dependency falls into. We've discussed this from time to time. Is it worth a mention?

Reply:
That's a good question. I would kind of like to make that a separate thing, outside of this chapter and PR.  I'm not sure where it would go, but I think it would be great to have written guidelines for dependencies. I'm wondering if there are other guidelines that are worth writing down? Like maybe review guidelines? Test requirements? Guidelines for when to revert?